src/app/components/Image/index.tsx

```diff
@@ -142,8 +142,8 @@ const Image = ({
               src={src}
               width={width}
               height={height}
-              srcSet={getImgSrcSet()}
-              sizes={getImgSizes()}
+              {...(srcSet && { srcSet: getImgSrcSet() })}
+              {...(getImgSizes() && { sizes: getImgSizes() })}
```

Comment:
Might we be best to assign sizes to a separate variable, instead of running `getImgSizes()` multiple times? 

e.g. `const sizes = getImgSizes();`

and then:
```suggestion
              sizes={sizes}
```